src/config/config.js

```diff
@@ -136,25 +140,23 @@ const config = convict({
   isProduction: {
     doc: 'If this application running in the production environment',
     format: Boolean,
-    default: process.env.NODE_ENV === 'production'
+    default: isProduction
   },
   isDevelopment: {
     doc: 'If this application running in the development environment',
     format: Boolean,
-    default: process.env.NODE_ENV !== 'production'
+    default: !isProduction
```

Comment:
if NODE_ENV === 'test' then `isDevelopment` will also be true - I know you haven't changed the logic here

Reply:
yeah theres a few other things incorrectly hooked into this. I'll have a look. At the time it blew up and so I kept it the same